src/fragments/Slider.react.js

```diff
@@ -20,12 +20,14 @@ export default class Slider extends Component {
         this._computeStyle = computeSliderStyle();
         this.state = {
             value: props.value,
+            drag_value: props.value,
```

Comment:
Does `drag_value` actually need to be included in `state`? I think that's what `this.state.value` already is...

Reply:
I removed it.